### DISCUSSION TITLE
HD300 wireless

### DISCUSSION THREAD

**Discussion by TGsell1** [OpenSageTV/Community#112] [SageTV Extenders (HD300,HD200)]
Can anyone help with installing a wireless dongle on an HD300? my HD300 ethernet port seems to have died (no flashing whatsoever) - has anyone experienced the same issue and fixed it with a wireless adapter? other functions seem to work fine - just need a network connection!

**Comment by JustFred51** [comment]
Others have been able to resolve the symptom you're experiencing.  See: [https://forums.sagetv.com/forums/showthread_t_61429.html?t=61429](https://forums.sagetv.com/forums/showthread_t_61429.html?t=61429)

**Comment by TGsell1** [comment]
Thanks. I tried this trick and it didn’t work primarily I think because I am not seeing flashing lights in the Ethernet port so for some reason it’s toast.   My only hope at this point is to try the wireless route with the compatible dongle I just found on eBay.  I just hate to give up on this one as I might need it someday.  Appreciate all the help Sent from my iPhoneOn Jan 25, 2024, at 9:27 PM, Keith Fischer ***@***.***> wrote:﻿
Others have been able to resolve the symptom you're experiencing.  See: https://forums.sagetv.com/forums/showthread_t_61429.html?t=61429

**Comment by TGsell1** [comment]
Exactly my plan!  Does anyone have a link for the firmware? Thanks allSent from my iPhoneOn Jan 26, 2024, at 7:17 AM, KeithAbbott ***@***.***> wrote:﻿
If you can get it connected via wifi, I would try reloading the firmware and see if that solves the Ethernet port problem.

Server Hardware: MSI Z270 SLI Plus ATX Motherboard, Intel i7-7700T CPU, 32GB Memory
Server Software: Unraid 6.12.6, sagetvopen-sagetv-server-opendct-java11 Docker (version 2.0.7)
Tuners: 2 x SiliconDust HDHomeRun Prime Cable TV Tuners, SiliconDust HDHomeRun CONNECT 4K OTA Tuner
Clients: Multiple HD300 Extenders, Multiple Fire TV Stick 4K Max w/MiniClient
Miscellaneous: Multiple Sony RM-VLZ620 Universal Remote Controls

**Comment by TGsell1** [comment]
Not sure I guess the latest non-beta and beta. ? Which do you advise. ? Thanks Keith!Sent from my iPhoneOn Jan 26, 2024, at 10:17 AM, KeithAbbott ***@***.***> wrote:﻿
I've saved most of the firmware versions.  Which version do you need?

Server Hardware: MSI Z270 SLI Plus ATX Motherboard, Intel i7-7700T CPU, 32GB Memory
Server Software: Unraid 6.12.6, sagetvopen-sagetv-server-opendct-java11 Docker (version 2.0.7)
Tuners: 2 x SiliconDust HDHomeRun Prime Cable TV Tuners, SiliconDust HDHomeRun CONNECT 4K OTA Tuner
Clients: Multiple HD300 Extenders, Multiple Fire TV Stick 4K Max w/MiniClient
Miscellaneous: Multiple Sony RM-VLZ620 Universal Remote Controls

**Comment by TGsell1** [comment]
Got ‘em!

**Comment by TGsell1** [comment]
I downloaded the firmware (thanks) on to a flash drive and inserted into one of the USB ports on another HD300 – how do I install the firmware from there?  system update seems to only check the internet... must be something dumb that I am missing..

**Comment by TGsell1** [comment]
Thanks again – I will try this.  Appreciate the support.

**Comment by TGsell1** [comment]
I was able to flash the firmware thanks! I am waiting for the Belkin wireless dongle from eBay to try to salvage this hd300.  I’ve given up hope that the Ethernet port will be functional again since I am not seeing flashing lights.  Only a brief dual flash during boot up then nada. It’s worth trying to keep these devices alive!thanksSent from my iPhoneOn Jan 26, 2024, at 10:44 AM, KeithAbbott ***@***.***> wrote:﻿
I think every version of firmware was considered beta, I don't think any of them ended up making it to stable.  Also, there is no single version that is the "best", each version has pros and cons.  I have settled on version 20101209-0 for my own environment, you can download it from https://drive.google.com/file/d/1fNosXQcCil76WYMUKHIg5wcXQsaL1eO9/view?usp=sharing
The link will be good until I remove it this Sunday.

Server Hardware: MSI Z270 SLI Plus ATX Motherboard, Intel i7-7700T CPU, 32GB Memory
Server Software: Unraid 6.12.6, sagetvopen-sagetv-server-opendct-java11 Docker (version 2.0.7)
Tuners: 2 x SiliconDust HDHomeRun Prime Cable TV Tuners, SiliconDust HDHomeRun CONNECT 4K OTA Tuner
Clients: Multiple HD300 Extenders, Multiple Fire TV Stick 4K Max w/MiniClient
Miscellaneous: Multiple Sony RM-VLZ620 Universal Remote Controls

**Comment by TGsell1** [comment]
Thanks to Keith I was able to resurrect my HD300 with the toasted ethernet port.   I ordered another one of those Belkin USB adapters since the HD300 must have drivers for it natively installed and for a couple of bucks its nice to have the wireless option in the future.   Another example that this forum can be kept useful and alive even if we miss the user friendliness of the old one.

**Comment by TGsell1** [comment]
Firmware update did not fix the port.  I was able to flash it so that could be a handy technique for the future.   The wireless adapter installed perfectly.  The knowledge of which (2?) adapters work with the HD300 is critical! Ty Sent from my iPhoneOn Jan 29, 2024, at 9:13 PM, KeithAbbott ***@***.***> wrote:﻿
Glad that you are back up and running!  Did reloading the firmware fix the ethernet port, or are you having to use the USB WiFi adapter?

Server Hardware: MSI Z270 SLI Plus ATX Motherboard, Intel i7-7700T CPU, 32GB Memory
Server Software: Unraid 6.12.6, sagetvopen-sagetv-server-opendct-java11 Docker (version 2.0.7)
Tuners: 2 x SiliconDust HDHomeRun Prime Cable TV Tuners, SiliconDust HDHomeRun CONNECT 4K OTA Tuner
Clients: Multiple HD300 Extenders, Multiple Fire TV Stick 4K Max w/MiniClient
Miscellaneous: Multiple Sony RM-VLZ620 Universal Remote Controls

**Reply by sircupp** [reply]
Do you have the make and model or a link to the wireless adapter.  I looking to purchase one.  I bought one previously but couldn't get it to work.

**Comment by TGsell1** [comment]
Yes.  And further I ordered two from eBay and one of the’versions’ that model worked and the other did not load despite many tries.   Anyway I’m happy with one.  Sent from my iPhoneOn Mar 16, 2024, at 2:14 PM, KeithAbbott ***@***.***> wrote:﻿
AFAIK, there are only two adapters that will work.  There was a SageTV branded adapter, and there was a Belkin F5D8051 Wireless Network USB Adapter.

Server Hardware: MSI Z270 SLI Plus ATX Motherboard, Intel i7-7700T CPU, 32GB Memory
Server Software: Unraid 6.12.8, sagetvopen-sagetv-server-opendct-java11 Docker (version 2.0.7)
Tuners: 2 x SiliconDust HDHomeRun Prime Cable TV Tuners, SiliconDust HDHomeRun CONNECT 4K OTA Tuner
Clients: Multiple HD300 Extenders, Multiple Fire TV Stick 4K Max w/MiniClient
Miscellaneous: Multiple Sony RM-VLZ620 Universal Remote Controls
